common/config.py

```diff
@@ -1,7 +1,18 @@
 """
 Application Config
 """
-FUND_STORE = (
-    "https://funding-service-design-fund-store-dev.london.cloudapps.digital"
+from os import getenv
+
+
+FUND_STORE = getenv(
```

Comment:
@srh-sloan If we wanted to include more microservices in the performance tests, we just need to do copy how you have done it for Fund and Application Store?